manager2/src/app/user/user.component.ts

```diff
@@ -255,8 +239,6 @@ export class UserComponent implements OnInit {
             new_projects.push({id: projects[i].id, owner: false, group: projects[i].group, member: false});
         }
     }
-    user_projects.sort(this._compareId)
-    new_projects.sort(this._compareId)
```

Comment:
why removal of sort on groups, user_projects and new_projects ?

Reply:
I think they are useless as those list as created from already ordered list. I can re-add it if you want ;)

Reply:
Are you sure they are ordered ?
Should not count anyway on order of API result (may change on backend).

Reply:
yes sort is done in the projects services in frontend:
https://github.com/genouest/genouestaccountmanager/pull/91/files#diff-fc06c10047d9960aa88b78aa835e45f8

Reply:
But as groups does not come from groups services (there are from user.group) I will re-add the sort for groups

Reply:
https://github.com/genouest/genouestaccountmanager/pull/91/commits/7743e72be3e6155f199fed854a7c21bbca76d067